package/com.unity.formats.usd/Runtime/Scripts/Timeline/UsdRecorderBehaviour.cs

```diff
@@ -82,6 +109,11 @@ public class UsdRecorderBehaviour : PlayableBehaviour {
 
         // Export the "default" frame, that is, all data which doesn't vary over time.
         Clip.UsdScene.Time = null;
+
+        // USDZ is in centimeters.
+        if (Clip.IsUSDZ)
+          root.transform.localScale = localScale * 100;
```

Comment:
Nit: add braces

More important: can we make the scale part of the ExportContext and handle it here?
https://github.com/Unity-Technologies/usd-unity-sdk/blob/fde5076004a8d4dc1e852adb3ffa334872dbaed1/package/com.unity.formats.usd/Runtime/Scripts/IO/Scene/SceneExporter.cs#L149

So that it's symetrical with the way we handle it for SceneImport:
https://github.com/Unity-Technologies/usd-unity-sdk/blob/fde5076004a8d4dc1e852adb3ffa334872dbaed1/package/com.unity.formats.usd/Runtime/Scripts/IO/Scene/SceneImportOptions.cs#L115

Reply:
That's definitely a better place.